integration-tests/originate-known-contracts.ts

```diff
@@ -7,28 +7,64 @@ import BigNumber from 'bignumber.js';
 import { codeViewsTopLevel } from './data/contract_views_top_level';
 import { knownBigMapContract } from './data/knownBigMapContract';
 import { knownContract } from './data/knownContract';
-import * as fs from 'fs';
+import * as fs from 'fs/promises';
 
 const MUTEZ_UNIT = new BigNumber(1000000);
 
 CONFIGS().forEach(({ lib, setup, protocol }) => {
   const tezos = lib;
   let keyPkh: string = "";
   let keyInitialBalance: BigNumber = new BigNumber(0);
+  let protocolShort = protocol.substring(0, 9);
 
   (async () => {
     await setup(true);
-    fs.writeFile(`known-contracts-${protocol.substring(0,9)}.ts`, '', (err: any) => {
-      if (err) {
-        console.error(err);
+
+    let outputFile = await fs.open(`known-contracts-${protocolShort}.ts`, 'w');
+    let writeOutput = async (line: string): Promise<void> => {
+      return fs
+        .writeFile(outputFile, line + '\n')
+        .catch((err: any) => {
+          console.error(err);
+        });
+    };
+    let appendOutput = async (line: string): Promise<void> => {
+      return fs
+        .appendFile(outputFile, line + '\n')
+        .catch((err: any) => {
+          console.error(err);
+        });
+    };
+    let originateKnownContract = async (contractName: string, tezos: TezosToolkit, contractOriginateParams: OriginateParams): Promise<void> => {
+      try {
+        const operation = await tezos.contract.originate(contractOriginateParams);
+        const contract = await operation.contract();
+        console.log(`known ${contractName} address:  ${contract.address}`);
+        console.log(`::set-output ${contractName}::${contract.address}\n`);
```

Comment:
This `::set-output` command needs a `name` parameter to fix the issue mentioned in https://github.com/ecadlabs/taquito/pull/2164#issuecomment-1355710953